components/Footer.js

```diff
@@ -214,7 +214,19 @@ export default function Footer() {
                 <Link
                   key={item.name}
                   href={item.href}
-                  className="text-primary-low-high hover:text-primary-low"
+                  className={
+                    item.name == "LinkedIn"
+                      ? "text-primary-low-high hover:text-sky-600"
+                      : item.name == "Twitter"
+                        ? "text-primary-low-high hover:text-black"
+                        : item.name == "Instagram"
+                          ? "text-primary-low-high hover:text-rose-600"
+                          : item.name == "GitHub"
+                            ? "text-primary-low-high hover:text-black"
+                            : item.name == "YouTube"
+                              ? "text-primary-low-high hover:text-rose-600"
+                              : ""
+                  }
```

Comment:
Avoid nesting ternary expressions, as it typically hinders readability. You can find a helpful article on this topic [here](https://www.sonarsource.com/blog/stop-nesting-ternaries-javascript/#:~:text=Nested%20ternaries%20are%20rarely%20clear,in%20one's%20head%20as%20possible.)

Here's a revised version of the code:
```javascript
const socialIconClassMappings = {
  "LinkedIn": "text-primary-low-high hover:text-sky-600",
  "Twitter": "text-primary-low-high hover:text-black",
  "Instagram": "text-primary-low-high hover:text-rose-600",
  "GitHub": "text-primary-low-high hover:text-black",
  "YouTube": "text-primary-low-high hover:text-rose-600",
};

const className = socialIconClassMappings[item.name] || "";
````
This approach improves clarity and maintainability by using a mapping object instead of nested ternaries.

Reply:
@ThomasCode92  I am beginner to opensource.  Thanks for pointing out my mistake.

Reply:
You're welcome!
It's not a mistake, your code will work. But using an object for mapping provides a clear and maintainable structure, making it easier to manage and extend the code. If a new social media channel needs to be added, it can be seamlessly incorporated into the object without the need to modify conditional statements. This approach promotes code simplicity and enhances maintainability.

Reply:
@ThomasCode92  Understood! it's needed for scalability.